### ISSUE TITLE
Accessibility/WCAG: automatic test flags textarea for missing label field

### ISSUE THREAD

**Issue by johanbove** [earthstar-project/letterbox#12]
The field has the placeholder "Supports markdown" but does not have a visible text label.

[WCAG rule](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#form_elements_should_have_a_visible_text_label)

![Screenshot 2022-04-15 083019](https://user-images.githubusercontent.com/922765/163531099-eba0711d-be69-40ff-acde-8a688d0e6d4f.png)

**Comment by johanbove** [comment]
The missing fields is also a problem reported on the "post" view.

I know Tailwind CSS has a "sr-only" class name which could be used on the label elements, so they would be there for screen-readers. However best practice is to always have the label of a field visible. Placeholders should not be used for this purpose.
